alma/alma-woocommerce-gateway
# Missing shipping information for In page

## Reason for change

When In Page is activated we loose the shipping informations + the client infos
[ClickUp task](https://linear.app/almapay/issue/MPP-573/in-page-frais-de-ports-manquants)


### How to test

- Activate In Page
- Create shipping zones
- Create an order with in page and a shipping option
- Verify in the BO the order, we must see the shipping option choosen


### Checklist for authors and reviewers
